Common/Util/LeanData.cs

```diff
@@ -204,6 +204,41 @@ public static string GenerateLine(IBaseData data, SecurityType securityType, Res
                     }
                     break;
 
+                case SecurityType.Index:
+                    switch (resolution)
+                    {
+                        case Resolution.Tick:
+                            var tick = data as Tick;
+                            if (tick == null)
+                            {
+                                throw new ArgumentException("Expected data of type 'Tick'", nameof(data));
+                            }
+                            return ToCsv(milliseconds, tick.BidPrice, tick.AskPrice);
+
+                        case Resolution.Second:
+                        case Resolution.Minute:
```

Comment:
We only have tradebars; so for `LeadData` and the `ParseIndex` it'll be trades only.